docker/build-tools/Dockerfile

```diff
@@ -539,11 +596,15 @@ WORKDIR /
 
 ENTRYPOINT ["/usr/local/bin/docker-entrypoint"]
 
+
 ##############
 # Clang+LLVM
 ##############
 
-FROM ubuntu:xenial as clang_context
+FROM ubuntu:xenial AS clang_context_amd64
+FROM ubuntu:focal AS clang_context_arm64
```

Comment:
nit: I would add a comment explaining the discrepancy here for the future

Reply:
Does this pull both xenial and focal image ?
Nit: If the answer is yes, can we avoid it?

Reply:
Nope, it is based on the final `--target` and `--platform` (by  `docker buildx build`).